css/style.css

```diff
@@ -0,0 +1,48 @@
+body{
+    margin: 0;
+    background-color: rgba(234, 225, 201, 1);
+    font-family: "Jaini Purva";
+}
+.jaini-purva-regular {
+    font-family: "Jaini Purva", system-ui;
+    font-weight: 400;
+    font-style: normal;
+  }
+  *{
+    box-sizing: border-box;
+  }
+  .column{
+    float: left;
+    width: 33.33%;
+    padding:10px;
+    height:fit-content
+  }
+  .row{
+    content: "";
```

Comment:
what is this for?

Reply:
Added something unnecessary, it's removed now.